src/en/clouds-maas-manual.md

```diff
@@ -0,0 +1,68 @@
+Title: Manually adding MAAS clouds
+
+# Manually adding MAAS clouds
+
+MAAS clouds may be registeres with juju using the interactive
+`add-cloud` command. If you wish to define these clouds in a YAML
+configuration file instead, this is also possible. The YAML file
+describing a MAAS cloud is straightforward - all
+that varies is the endpoint and the name, since they use the same
+authentication method. For example, below is the file maas-clouds.yaml:
+ 
+```yaml
+clouds:
+   devmaas:
+      type: maas
+      auth-types: [oauth1]
+      endpoint: http://devmaas/MAAS
+   testmaas:
+      type: maas
+      auth-types: [oauth1]
+      endpoint: http://172.18.42.10/MAAS
+   prodmaas:
+      type: maas
+      auth-types: [oauth1]
+      endpoint: http://prodmaas/MAAS
+```
+
+This defines three MAAS (region) controllers. To add a MAAS cloud from this
```

Comment:
This example YAML defines...
To use this file to add a MAAS cloud to Juju, use: